### DISCUSSION TITLE
Add toggle for Camera Avoidance on phones.

### DISCUSSION THREAD

**Discussion by SoffiMeower** [ppy/osu#20970] [Ideas] [answered]
From what I've noticed, osu!lazer seems to try to avoid the portion of the screen that has an embedded front camera.
![Screenshot_20221027-084748.png](https://user-images.githubusercontent.com/45863583/198290788-d648452f-90b4-4b06-b20d-bca0150d2b88.png)
I think there should be an option to disable this setting, as a lot of Android phones have a small pinhole (Small circle) camera on the corners, not really obstructing gameplay. Along with that, the touchscreen usually works on the camera area due to how small the camera is.
![motorola-moto-g-stylus-5g-1-1536x1524.jpg](https://user-images.githubusercontent.com/45863583/198291399-8025b41a-d011-4672-89e3-c2294b355ee8.jpg)
(Ref Pic of my phone to show the camera on the screen)
The camera is tiny, and yet osu!lazer removes a good chunk of the screen, making it look quite... Odd.

**Comment by integerrr** [comment, marked as answer]
https://github.com/ppy/osu/discussions?discussions_q=notch

**Reply by peppy** [reply]
Will just add here that I think having a toggle for this as a first step may be best. I don't foresee us having better UI support for it in the near future.

**Reply by peppy** [reply]
Added via #20983.
